lib/ecto/query.ex

```diff
@@ -475,11 +475,34 @@ defmodule Ecto.Query do
   @doc """
   Resets a previously set field on a query.
 
-  It can reset any query field except the query source (`from`).
+  It can reset many fields except the query source (`from`).
 
   ## Example
 
-      query |> Ecto.Query.exclude(:select)
+      Ecto.Query.exclude(query, :join)
+      Ecto.Query.exclude(query, :where)
+      Ecto.Query.exclude(query, :order_by)
+      Ecto.Query.exclude(query, :group_by)
+      Ecto.Query.exclude(query, :having)
+      Ecto.Query.exclude(query, :distinct)
+      Ecto.Query.exclude(query, :select)
+      Ecto.Query.exclude(query, :limit)
+      Ecto.Query.exclude(query, :offset)
+      Ecto.Query.exclude(query, :lock)
+      Ecto.Query.exclude(query, :preload)
+
+  You can also remove specific joins as well such as `left_join` and
+  `inner_join`.
+
+  ## Example
```

Comment:
You don't need this other exemple header in here.

Reply:
removed